eo-runtime/src/test/eo/org/eolang/rust-tests.eo

```diff
@@ -307,8 +308,8 @@
     use eo_env::eo_enum::EO;
     use eo_env::eo_enum::EO::{EOError};
 
-    pub fn foo(_env: &mut EOEnv) -> EO {
-      EOError("Custom error".to_string())
+    pub fn foo(_env: &mut EOEnv) -> Option<EO> {
+      Some(EOError("Custom error".to_string()))
```

Comment:
@levBagryansky Should there also be 4 spaces missing at the beginning of the line with the code?